crypto/evp/m_sm3.c

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
This file could be placed in `crypto/sm3` just as well.  As a matter of fact, I'd like to encourage that.  Have a look at `crypto/blake2/` for inspiration.

Reply:
I agree.